.gitignore

```diff
@@ -6,6 +6,7 @@ generic.key
 series.txt
 series_sorted.txt
 src/
+lists/*
```

Comment:
Since we're adding lists to .gitignore, do you think it's a good idea to just delete the directory from the repo?